addons/website_theme_install/models/theme_models.py

```diff
@@ -212,6 +212,13 @@ class IrUiView(models.Model):
 
     theme_template_id = fields.Many2one('theme.ir.ui.view')
 
+    def write(self, vals):
+        # Do not mark the view as user updated if original view arch is similar.
+        if self.theme_template_id and (('arch' in vals and self.theme_template_id.arch == vals['arch'])
```

Comment:
```suggestion
    arch = vals.get('arch', vals.get('arch_db'))
    if self.theme_template_id and and arch == self.theme_template_id.arch:
        vals['arch_updated'] = False
```

Fix look good to me, what do you thing about this small diff, no changes, just more readable imo ?

Reply:
Yes much more readable.